package/src/prmon.cpp

```diff
@@ -219,26 +219,29 @@ int main(int argc, char* argv[]) {
   const char* default_filename = "prmon.txt";
   const char* default_json_summary = "prmon.json";
   const unsigned int default_interval = 30;
+  const bool default_store_cpu_freq = false;
 
   pid_t pid = -1;
   bool got_pid = false;
   std::string filename{default_filename};
   std::string jsonSummary{default_json_summary};
   std::vector<std::string> netdevs{};
   unsigned int interval{default_interval};
+  unsigned int store_cpu_freq{default_store_cpu_freq};
   int do_help{0};
 
   static struct option long_options[] = {
       {"pid", required_argument, NULL, 'p'},
       {"filename", required_argument, NULL, 'f'},
       {"json-summary", required_argument, NULL, 'j'},
       {"interval", required_argument, NULL, 'i'},
+      {"store-cpu-freq", no_argument, NULL, 's'},
       {"netdev", required_argument, NULL, 'n'},
       {"help", no_argument, NULL, 'h'},
       {0, 0, 0, 0}};
 
   char c;
-  while ((c = getopt_long(argc, argv, "p:f:j:i:n:h", long_options, NULL)) !=
+  while ((c = getopt_long(argc, argv, "p:f:j:i:s:n:h", long_options, NULL)) !=
```

Comment:
This should just be `s` and not `s:`, which indicates that the short option should take an argument